roles/KubevirtCommonTemplatesBundle/defaults/main.yml

```diff
@@ -1,3 +1,4 @@
 ---
 # defaults file for KubevirtCommonTemplatesBundle
 version: "{{ templates_version | default('latest') }}"
+base_image_namespace: "{{ meta.namespace }}-golden-image"
```

Comment:
please put this variable in `_defaults.yml ` file in the root of ssp operator. Set this variable as `hardcoded` because we will have to anyway replace this variable in D/S